docs/code-reviews/pull-requests.md

```diff
@@ -75,6 +86,5 @@ See also [Pull Request Template](pull-request-template/pull-request-template.md)
 * [Google approach to PR size](https://google.github.io/eng-practices/review/developer/small-cls.html)
 * [Feature Flags](https://www.martinfowler.com/articles/feature-toggles.html)
 * [Facebook approach to hidden features](https://launchdarkly.com/blog/secret-to-facebooks-hacker-engineering-culture/)
```

Comment:
```suggestion
* [Facebook approach to hidden features](https://launchdarkly.com/blog/secret-to-facebooks-hacker-engineering-culture/)
* [Azure approach to canary releases](https://docs.microsoft.com/en-us/azure/architecture/framework/devops/deployment#stage-your-workloads)
```

Reply:
I have fixed this in #775 - so that it points to the new correct location -- once this PR is ready for merge, let's merge in that change instead of removing this content